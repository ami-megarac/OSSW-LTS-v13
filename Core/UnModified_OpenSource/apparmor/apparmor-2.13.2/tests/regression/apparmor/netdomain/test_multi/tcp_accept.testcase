send_ip=127.0.0.1
send_port=8989
send_profile=tcp_connect to 127.0.0.1:8990 from 127.0.0.1:8989,
receive_ip=127.0.0.1
receive_port=8990
receive_profile=tcp_accept from 127.0.0.1:8989 to 127.0.0.1:8990,
message=tcp_accept pass
proto=tcp
