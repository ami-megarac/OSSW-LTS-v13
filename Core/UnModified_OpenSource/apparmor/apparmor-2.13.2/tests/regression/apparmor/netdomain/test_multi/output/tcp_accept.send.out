Sending "tcp_accept pass"
